eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -82,6 +87,9 @@ public FakeMaven withProgram(final String... program) throws IOException {
      * Set default the params for all mojos.
      *
      * @return The same maven instance.
+     * @todo #1479:90min The method withDefaults() has to be called right in the exec() method.
```

Comment:
@volodya-lombrozo We estimate all puzzles with 30mins... It's agreed marker estimation for average dev task. Other puzzles if required will be created during development.

Reply:
@mximp let's discuss it [separately](https://github.com/objectionary/eo/issues/1486#issue-1463458218).